composer.json

```diff
@@ -40,7 +38,9 @@
         "aws/aws-sdk-php": "~2.7",
         "doctrine/mongodb-odm": "~1.0",
         "jackalope/jackalope-doctrine-dbal": "~1.1",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/phpunit-bridge": "~2.7|~3.0",
+        "friendsofsymfony/rest-bundle": "~1.1",
```

Comment:
hmm, but for the development it looks weird, so on my local machine a would have an api with docs and on prod not... :-1: